src/wp-admin/includes/theme.php

```diff
@@ -966,10 +985,10 @@ function customize_themes_print_templates() {
 							<# } else if ( ! data.compatibleWP ) { #>
 								<?php
 								_e( 'This theme does not work with your version of ClassicPress.' );
-								if ( current_user_can( 'update_core' ) ) {
+								if ( current_user_can( 'update_core' ) && $cp_needs_update ) {
 									printf(
 										/* translators: %s: URL to WordPress Updates screen. */
-										' ' . __( '<a href="%s">Please update ClassicPress</a>.' ),
+										' ' . __( '<a href="%s">Please cd .Press</a>.' ),
```

Comment:
This change doesn't look like it's intentional.

Reply:
Seems like I was changing directory...